core/src/main/web/app/helpers/helper.js

```diff
@@ -679,6 +679,9 @@
       cancelTimeout: function(promise) {
         return bkUtils.cancelTimeout(promise);
       },
+      trustAsHtml: function(piece) {
```

Comment:
Yeah I agree. This stuff is already trusted, it's coming from us. We don't need to clean it.